core/gdb_console.c

```diff
@@ -21,23 +21,22 @@
 #include <core/gdb_console.h>
 
 
-static void writeStringToExclusiveGdbCommChannel(const char* pString);
 void WriteStringToGdbConsole(const char* pString)
 {
-    writeStringToExclusiveGdbCommChannel(pString);
+    WriteSizeStringToGdbConsole(pString, mri_strlen(pString));
```

Comment:
Doesn't compile! **gdb_console.c** is missing a `#include <core/libc.h>` to declare mri_strlen().

Reply:
Done.